pkg/metrics/wal/wal_test.go

```diff
@@ -163,12 +164,16 @@ func TestStorage_DuplicateExemplarsIgnored(t *testing.T) {
 	_, _ = app.AppendExemplar(sRef, nil, e)
 	_, _ = app.AppendExemplar(sRef, nil, e)
 
+	e.Ts = 24
+	_, _ = app.AppendExemplar(sRef, nil, e)
+	_, _ = app.AppendExemplar(sRef, nil, e)
+
```

Comment:
Adding an OOO exemplar doesn't change the number of exemplars stored in line 177.